configs/hand/gesture_sview_rgbd_vid/mtut/nvgesture/i3d_nvgesture_bbox_112x112_fps15_rgb.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+_base_ = [
+    '../../../../_base_/default_runtime.py',
+    '../../../../_base_/datasets/nvgesture.py'
+]
+
+checkpoint_config = dict(interval=5)
+evaluation = dict(interval=5, metric='AP', save_best='AP_rgb')
+
+optimizer = dict(
+    type='SGD',
+    lr=1e-1,
+    momentum=0.9,
+)
+optimizer_config = dict(grad_clip=None)
+# learning policy
+lr_config = dict(policy='step', gamma=0.1, step=[30, 60, 90, 110])
+total_epochs = 130
+log_config = dict(interval=10)
+
+custom_hooks_config = [dict(type='ModelSetEpochHook')]
+
+model = dict(
+    type='GestureRecognizer',
+    modality=['rgb'],
```

Comment:
Is this config for rgb only? 
Then the config file path `gesture_sview_rgbd_vid`  is a little bit confusing.

Reply:
Yes. As stated in `/configs/hand/gesture_sview_rgbd_vid/mtut/nvgesture/i3d_nvgesture.md`, MTUT supports multi-modal training and uni-modal inference. This config is mainly used for demo and webcam with RGB videos.